library/linux/Makefile

```diff
@@ -11,6 +11,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# ================================== Variables ==================================
```

Comment:
I'd prefer not to make purely stylistic changes to the Makefiles, especially when they aren't being done consistently across the project.

Reply:
Yes, you're right...